packages/york-react-native/src/components/Picker/index.js

```diff
@@ -0,0 +1,287 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  StyleSheet,
+  TouchableOpacity,
+  TouchableHighlight,
+  TouchableWithoutFeedback,
+  Modal,
+  View,
+  ScrollView,
+  Image,
+  Platform,
+  Dimensions,
+  Keyboard,
+} from 'react-native'
+import * as R from 'ramda'
+import { colors } from '@qlean/york-core'
+
+import Text from 'york-react-native/components/Text'
+import Separator from 'york-react-native/components/Separator'
+
+import { uiPoint, sizes } from 'york-react-native/utils/styles'
+
+const pickerPaddingTop = sizes[12]
+const pickerPaddingBottom = sizes[16]
+const pickerPaddignHorizontal = sizes[6]
+const pickerItemHeight = uiPoint * 11
+const pickerContentPaddingVertical = sizes[2]
+
+const { width: screenWidth, height: screenHeight } = Dimensions.get('window')
+
+/**
+ * Dimensions возвращает полную высоту экрана, поэтому на Андроиде нужно компенсировать высоту
+ * статус бара. Она не всегда равна точно 25, но расхождение в пару пунктов крайне незначительно
+ * влияет на общие пропорции модального окна.
+ * */
+const statusBarHeight = Platform.OS === 'android' ? 25 : 0
```

Comment:
https://stackoverflow.com/questions/35436643/how-to-find-height-of-status-bar-in-android-through-react-native